src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -652,11 +666,19 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _receivedInstanceOfflinePing =
                 LoggerMessage.Define<string>(LogLevel.Information, new EventId(31, "ReceivedInstanceOfflinePing"), "Received instance offline service ping: {InstanceId}");
 
+            private static readonly Action<ILogger, Exception> _authorizeFailed =
+                LoggerMessage.Define(LogLevel.Error, new EventId(32, "AuthorizeFailed"), "Authorize failed");
```

Comment:
Is Warning enough? What happens when auth failed?

Reply:
Error generally means customer should do something to prevent it

Reply:
The server connection will be dropped if the auth failed.

In management SDK, the accessKey will lose the ability to sign tokens.

The customer should find out why he doesn't have permission over the resource.